src/Lexer.ts

```diff
@@ -1,12 +1,13 @@
-import { Tokenizer } from './Tokenizer.js';
-import { defaults } from './defaults.js';
-import { block, inline } from './rules.js';
+import { _Tokenizer } from './Tokenizer.ts';
```

Comment:
I thought the `.ts` extension wasn't supported and it had to use `.js` (like you are doing in the Instance.ts file already)

Is this a tsup feature?

Reply:
It's a typescript feature [`allowImportingTsExtensions`](https://www.typescriptlang.org/tsconfig#allowImportingTsExtensions) and esbuild, which is used for the actual transpilation in tsup, also supports it during bundling https://esbuild.github.io/content-types/#default-interop

Reply:
I see. Any reason why the `Instance.ts` file is using `.js` imports?

Reply:
I just missed to adapt them during the rebase. Fixed now